keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/titus/batch/ContainerJobConfig.kt

```diff
@@ -1,123 +1,112 @@
 package com.netflix.spinnaker.keel.titus.batch
 
-import com.netflix.spinnaker.keel.titus.batch.ImageVersionReference.*
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.keel.api.ec2.Capacity
+import com.netflix.spinnaker.keel.api.titus.TitusServerGroup
+import java.time.Duration
 
-const val RUN_JOB_TYPE : String = "runJob"
-
-/**
- * Containers can be identified by either:
- *  tag (e.g., "latest")
- *  digest (e.g., "sha256:780f11bfc03495da29f9e2d25bf55123330715fb494ac27f45c96f808fd2d4c5")
- */
-enum class ImageVersionReference {
-  TAG,
-  DIGEST
-}
+const val RUN_JOB_TYPE: String = "runJob"
 
 /**
  * Holds the config info needed to execute a batch job on Titus via the Run Job stage
  */
 data class ContainerJobConfig(
-  val name : String,
-  val application: String,
-  val account: String,
-  val region: String,
-  val organization: String,
   /**
    * Repository name associated with the container, e.g.: "acme/widget"
    */
   val repository: String,
+  val tag: String? = "latest",
+  val digest: String? = null,
+  val application: String,
+  val location: TitusServerGroup.Location,
+  val resources: TitusServerGroup.Resources = TitusServerGroup.Resources(
+    cpu = 2,
+    memory = 2048,
+    disk = 256,
+    networkMbps = 1024
+  ),
+  val capacity: Capacity = Capacity(min = 1, max = 1, desired = 1),
   val serviceAccount: String,
   val credentials: String,
-  val registry : String,
-  val type: ImageVersionReference = TAG,
-  /**
-   * Either a tag (e.g., "latest") or a digest (e.g., "sha256:be93efc727ba59813adc896859bccf32fb0f02202fe0526a9cd76326b9729cb3),
-   * depending on what [type] is
-   */
-  val versionIdentifier: String = "latest",
-  /**
-   * ID that identifies an image, e.g.:
-   *
-   * acme/widget:latest
-   * acme/widget:sha256:780f11bfc03495da29f9e2d25bf55123330715fb494ac27f45c96f808fd2d4c5
-   */
-  val imageId: String = "$repository:$versionIdentifier",
-  val desiredCapacity: Int= 1,
-  val maxCapacity: Int = 1,
-  val minCapacity: Int = 1,
-  val cpus: Int = 2,
-  val memoryMb: Int = 2048,
-  val diskMb: Int = 256,
-  val networkMbps: Int = 1024,
-  val gpus: Int = 0,
   val entrypoint: String = "",
-  val runtimeLimitSeconds: Int = 2700,
+  val runtimeLimit: Duration = Duration.ofSeconds(2700),
   val retries: Int = 0,
-  val cloudProvider : String = "titus",
-  val cloudProviderType: String = "aws",
```

Comment:
Again these fields don't seem to be necessary for the job to work. I think they're there because Orca offers other ways to run a job than Titus